doc/nrf/links.txt

```diff
@@ -74,6 +74,12 @@
 
 .. _`nRF Connect for Desktop`: https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Connect-for-desktop
 
+.. _`nRF Connect for Mobile`: https://www.nordicsemi.com/?sc_itemid=%7B41FF7A0B-B565-420A-95B7-B32122B5D3AD%7D
```

Comment:
It works, but https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Connect-for-mobile seems safer. ;)

Reply:
fixed